src/app/psat/modify-conditions/modify-conditions.component.html

```diff
@@ -19,9 +19,37 @@ <h3>Baseline</h3>
       </div>
     </div>
 
+    <!-- Modifications Start / Compare Two Pumps as baseline -->
```

Comment:
This may be what's causing some of the form display issues. You have two modification blocks where `psat.modifications[modificationIndex].psat.whatIfScenario == false` is being checked. You only need one modification block, and that property doesn't need to be checked. See my example branch